### DISCUSSION TITLE
How to add my colllege in GitHub India Externship

### DISCUSSION THREAD

**Discussion by afrid18** [github/india#611] [Students]
Hello, 
I am a student of [Sri Venkateswara College of Engineering and Technology](https://svcetedu.org/), and I would like to participate in GitHub India Externship. May I know how my college can be listed in the college listing?
Thanks 😀
Hello @mvkaran can you please tell me how?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by dgyani** [comment]
> 

Application process to become a Campus Program school is easy and quick. Faculty members / Head of Department or IT Heads can apply for the program [online here](https://education.github.com/schools)
 
GitHub Campus program offers immense benefits to students, teachers and the Institute and is completely free. Keeping in mind the interest in GitHub Externship, we are processing the applications at the earliest, so that students do not miss participating in this edition of GitHub Externship.

**Reply by afrid18** [reply]
Thanks @dgyani for your help. 😃

**Reply by afrid18** [reply]
@dgyani, will my school be listed [here](https://github.com/github/india/blob/main/Students/GCP-India.md), If our Head of Department apply for the program?

**Reply by dgyani** [reply]
Yes!

**Reply by afrid18** [reply]
Thanks a lot sir!
